.github/workflows/pre-commit.yml

```
@@ -0,0 +1,17 @@
+---
+name: pre-commit
+
+on:  # yamllint disable-line
```

nonblocking, just nice to not have to disable linter

```suggestion
"on":
```